src/quo2/components/settings/data_item/view.cljs

```diff
@@ -0,0 +1,141 @@
+(ns quo2.components.settings.data-item.view
+  (:require [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.settings.data-item.style :as style]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.common.not-implemented.view :as not-implemented]
+            [quo2.components.icon :as icons]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.list-items.preview-list :as preview-list]
+            [quo2.foundations.resources :as resources]
+            [quo2.components.avatars.account-avatar.view :as account-avatar]
+            [utils.i18n :as i18n]))
+
+(defn- left-loading
+  [{:keys [size blur? theme]}]
+  [rn/view {:style (style/loading-container size blur? theme)}])
+
+(defn- left-subtitle
+  [{:keys [theme size description icon icon-color blur? subtitle emoji-color emoji]}]
+  (let [background-color (colors/theme-colors
+                          (colors/custom-color emoji-color 50)
+                          (colors/custom-color emoji-color 60)
+                          theme)]
+    [rn/view {:style style/subtitle-container}
+     (when (not= :small size)
+       [rn/view {:style (style/subtitle-icon-container description)}
+        (case description
+          :icon    [icons/icon icon
+                    {:accessibility-label :description-icon
+                     :size                16
+                     :color               icon-color}]
+          :account [account-avatar/view
+                    {:customization-color background-color
+                     :size                16
+                     :emoji               emoji
+                     :type                :defaul}]
+          :network [rn/image
+                    {:accessibility-label :description-image
+                     :source              (resources/tokens :eth)
+                     :style               style/image}]
+          :default nil
+          nil)])
```

Comment:
You can remove `:default` for this `case`.

if you remove it and someone passes `:default` it will return the default case added (`nil`)

Reply:
Nice Catch.

Reply:
Done